test/test_images/recordevents/main.go

```diff
@@ -113,28 +115,34 @@ func (er *eventRecorder) handleGetEntry(w http.ResponseWriter, r *http.Request)
 	w.Write(entryBytes)
 }
 
-// handler for cloudevents
-func (er *eventRecorder) handler(ctx context.Context, event cloudevents.Event) {
-	cloudevents.HTTPTransportContextFrom(ctx)
+func (er *eventRecorder) handle(writer http.ResponseWriter, request *http.Request) {
+	m := cloudeventshttp.NewMessageFromHttpRequest(request)
+	defer m.Finish(nil)
 
-	tx := cloudevents.HTTPTransportContextFrom(ctx)
+	event, eventErr := cloudeventsbindings.ToEvent(context.TODO(), m)
```

Comment:
eventErr is an error when performing a conversion, not a validation error (like you seem to expect in `StoreEvent` signature)

Reply:
To get a validation error, you still need to invoke `event.Validate()`